### DISCUSSION TITLE
Android app not hiding nav bar

### DISCUSSION THREAD

**Discussion by jonnylitten** [redphx/better-xcloud#379] [General]
I am running Android 10 on my Nintendo Switch using Switchroot, and the navigation bar does not get hidden in the new better xcloud Android app

**Comment by KingNothing81** [comment]
You are in the discussion area. 
I think it would be better to file a bug on the Android's app GitHub page.

https://github.com/redphx/better-xcloud-android/issues

And before @redphx might be able to fix your problem, maybe you want to have a look at the other Android app that showed up here recently. It doesn't use a tool bar.

https://github.com/redphx/better-xcloud/discussions/363
